shared/chat/conversation/messages/wrapper/shared.js

```diff
@@ -57,7 +57,7 @@ const MessageWrapper = (props: Props) => (
         <Username includeHeader={props.includeHeader} author={props.author} isYou={props.isYou} isFollowing={props.isFollowing} isBroken={props.isBroken} />
         <Box style={_textContainerStyle} className='message' data-message-key={props.messageKey}>
           <Box style={_flexOneColumn}>
-            <props.innerClass messageKey={props.messageKey} onAction={props.onAction} />
+            <props.innerClass messageKey={props.messageKey} measure={props.measure} onAction={props.onAction} />
```

Comment:
This was causing `measure()` to not be called by the individual text/attachment components when their props updated. Might have caused some sizing issues?